Lib/test/test_cmd_line.py

```diff
@@ -541,31 +541,47 @@ def test_xdev(self):
         code = ("import sys, warnings; "
                 "print(' '.join('%s::%s' % (f[0], f[2].__name__) "
                                 "for f in warnings.filters))")
+        if sys.flags.debug:
+            resource_action = "default"
+        else:
+            resource_action = "ignore"
 
         out = self.run_xdev("-c", code)
         self.assertEqual(out,
+                         "default::Warning "
```

Comment:
default::Warning must come after BytesWarning. See also the discussion on https://bugs.python.org/issue32230

Reply:
I'll add a comment pointing out that `Py_BytesWarningFlag` means this filter is pretty redundant when neither `-b` nor `-bb` is passed.